### ISSUE TITLE
Reinstall docker container

### ISSUE THREAD

**Issue by CordlessWool** [solidnerd/docker-bookstack#10]
I set up this docker container behind a nginx container. The nginx only redirect calls to the apache.
I want to test if I could replace the bookstack container with a new one, for later updates.
The database already exist and login prozess is started, but server runs to an Internel Server error.

**Comment by solidnerd** [comment]
Hey, 
could you provide more information. Could you post you `docker-compose.yml` without any credentials. I hope that I understand it correct do you want to share with two bookstack container a single database ? This will not work. What you need to do for an upgrade is to shutdown your bookstack container, update your image version and start a new bookstack container.

**Comment by CordlessWool** [comment]
Hi

the first conainter was removed before I started the secound one. So there was only one bookstack container running. 

i added bookstack to two networks. One to connect to mysql database and one to an nginx proxy.
The docker command with option i used was:

docker run --name bookstack \
--network mysql_network \
-e DB_HOST=mysql_ip \
-e DB_DATABASE=bookstack_schema \
-e DB_USERNAME=bookstack_dbuser \
-e DB_PASSWORD=secret \
--restart=always \
solidnerd/bookstack

Afterwards I added it ti the nginx network with a fix IP.
